tapdb/sqlc/queries/addrs.sql

```diff
@@ -105,6 +105,32 @@ LEFT JOIN internal_keys
        ON managed_utxos.internal_key_id = internal_keys.key_id
 WHERE id = $1;
 
+-- name: FetchAddrEventByAddrKeyAndOutpoint :one
+WITH target_addr(addr_id) AS (
+    SELECT id
+    FROM addrs
+    WHERE addrs.taproot_output_key = $1
+)
+SELECT
+    addr_events.id, creation_time, status, asset_proof_id, asset_id,
```

Comment:
Why expose `addr_events.id` here? I didn't spot a use of it later on (may be in some follow-on PR?)

Reply:
It's part of the `address.Event` struct. It looks like it's currently only used in unit tests... Didn't want to also refactor them, as the PR was already growing quite a bit. But it looks like we could probably un-export the `address.Event.ID` field.